installscripts/cookbooks/jenkins/attributes/default.rb

```diff
@@ -25,6 +25,8 @@
 default['bbpassword'] = 'REPLACEME'
 default['sonaruser'] = 'REPLACEME'
 default['sonarpassword'] = 'REPLACEME'
+default['acluser'] = 'REPLACEME'
```

Comment:
`acluster` is little confusing. Can we use `acl_db_cluster` & `acl_db_password`?

Reply:
Done